[ID] Update and retranslate `Users with unique titles` (#5622)
ppy/osu-wiki: wiki/People/Users_with_unique_titles/id.md

# Daftar pengguna dengan gelar khusus

> Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [gugus pengguna (*user group*)](/wiki/People/The_Team#user-groups). Meskipun demikian, terdapat beberapa pengguna yang secara khusus dianugerahi gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang telah mereka torehkan.

Reviewer: ```suggestion
Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [gugus pengguna (*user group*)](/wiki/People/The_Team#gugus-pengguna). Meskipun demikian, terdapat beberapa pengguna yang secara khusus dianugerahi gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang telah mereka torehkan.
```

Reviewer: the id page is changing and should be redirected 
![image](https://user-images.githubusercontent.com/39128490/120093143-76cbc480-c142-11eb-9fa7-6e007af628b2.png)

Author: missed this somehow whoops, tyty

> Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [gugus pengguna (*user group*)](/wiki/People/The_Team#user-groups). Meskipun demikian, terdapat beberapa pengguna yang secara khusus dianugerahi gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang telah mereka torehkan.

Reviewer 2: ```suggestion
Pada umumnya, gelar yang disandang oleh seorang pengguna mencerminkan status keanggotaan mereka dalam suatu [grup tim (*user group*)](/wiki/People/The_Team#user-groups). Meskipun demikian, terdapat beberapa pengguna yang secara khusus diberikan gelar-gelar tertentu dengan berbagai alasan mulai dari prestasi, kontribusi, hingga momen-momen unik yang pernah mereka lakukan.
```

Reviewer: it should redirect to group member instead. so `gugus pengguna` is correct here.

Author: ya famoss is correct here; the current sentence is correct as is and changing it to `grup tim` would make it misleading instead

## Berdasarkan kategori, beserta alasannya

### Pemenang World Cup

> Setiap tahunnya, osu! menganugerahkan para pemenang turnamen [World Cup](/wiki/Tournaments#official-world-cups) dengan gelar khusus yang berhak untuk mereka sandang selama kurang lebih satu tahun hingga penyelenggaraan turnamen World Cup berikutnya.

Reviewer 2: ```suggestion
 Setiap tahunnya, osu! memberikan para pemenang turnamen [World Cup](/wiki/Tournaments#official-world-cups) dengan gelar khusus yang dapat mereka miliki selama kurang lebih satu tahun hingga penyelenggaraan turnamen World Cup berikutnya.
```

> Setiap tahunnya, osu! menganugerahkan para pemenang turnamen [World Cup](/wiki/Tournaments#official-world-cups) dengan gelar khusus yang berhak untuk mereka sandang selama kurang lebih satu tahun hingga penyelenggaraan turnamen World Cup berikutnya.

Reviewer 3: ```suggestion
Setiap tahunnya, osu! menganugerahkan para pemenang turnamen [World Cup](/wiki/Tournaments#official-world-cups) dengan gelar khusus yang berhak untuk mereka sandangi selama kurang lebih satu tahun hingga penyelenggaraan turnamen World Cup berikutnya.
```

> Setiap tahunnya, osu! menganugerahkan para pemenang turnamen [World Cup](/wiki/Tournaments#official-world-cups) dengan gelar khusus yang berhak untuk mereka sandang selama kurang lebih satu tahun hingga penyelenggaraan turnamen World Cup berikutnya.

Reviewer 3: ```suggestion
Setiap tahunnya, osu! menganugerahkan para pemenang turnamen [World Cup](/wiki/Tournaments#official-world-cups) dengan gelar khusus yang berhak untuk mereka sandangi selama kurang lebih satu tahun hingga penyelenggaraan turnamen World Cup berikutnya.
```

> Berikut merupakan para pemenang [OWC 2020](/wiki/Tournaments/OWC/2020) yang saat ini menyandang gelar **osu! Champion**:

Reviewer 2: ```suggestion
Berikut-berikut di bawah ini merupakan para pemenang [OWC 2020](/wiki/Tournaments/OWC/2020) yang saat ini memiliki gelar **osu! Champion**:
```

Author: (this applies to your other suggestions that are similar to this btw)

what you're suggesting atm, `Berikut-berikut di bawah ini`, grammatically speaking is actually ***very*** wrong. in english what you're suggesting is equivalent to `The followings below names are the winners...` which just doesn't make proper sense altogether.

`Berikut merupakan para pemenang` is already correct as is as it corresponds to the much more plausible `The following are the winners of...`.

Reviewer 2: yeah, kinda missed about that there

- ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296)
- ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150)

> Berikut merupakan para pemenang [TWC 2021](/wiki/Tournaments/TWC/2021) yang saat ini menyandang gelar **osu!taiko Champion**:

Reviewer 2: ```suggestion
Berikut-berikut di bawah ini merupakan para pemenang [TWC 2021](/wiki/Tournaments/TWC/2021) yang saat ini memiliki gelar **osu!taiko Champion**:
```

- ![][flag_JP] [ekumea1123](https://osu.ppy.sh/users/9119501)
- ![][flag_JP] [Grape\_Tea](https://osu.ppy.sh/users/9540073)
- ![][flag_JP] [Seren58](https://osu.ppy.sh/users/15252950)
- ![][flag_JP] [syaron105](https://osu.ppy.sh/users/8741695)
- ![][flag_JP] [uone](https://osu.ppy.sh/users/5321719)

> Berikut merupakan para pemenang [CWC 2020](/wiki/Tournaments/CWC/2020) yang saat ini menyandang gelar **osu!catch Champion**:

Reviewer 2: ```suggestion
Berikut-berikut di bawah ini merupakan para pemenang [CWC 2020](/wiki/Tournaments/CWC/2020) yang saat ini memiliki gelar **osu!catch Champion**:
```

- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)
- ![][flag_KR] [T s u m i](https://osu.ppy.sh/users/4080520)

> Berikut merupakan para pemenang [MWC 4K 2020](/wiki/Tournaments/MWC/2020_4K) yang saat ini menyandang gelar **osu!mania Champion**:

Reviewer 2: ```suggestion
Berikut-berikut di bawah inimerupakan para pemenang [MWC 4K 2020](/wiki/Tournaments/MWC/2020_4K) yang saat ini memiliki gelar **osu!mania Champion**:
```

- ![][flag_BR] [Remu](https://osu.ppy.sh/users/8700468)
- ![][flag_BR] [SillyFangirl](https://osu.ppy.sh/users/2288363)

### Pemenang kontes-kontes mapping resmi

> osu! pada umumnya menganugerahkan para pemenang kontes-kontes mapping yang berstatus resmi (*official mapping contests*) dengan gelar **Elite Mapper**.

Reviewer 2: ```suggestion
osu! pada umumnya memberikan para pemenang kontes-kontes mapping yang berstatus resmi (*official mapping contests*) dengan gelar **Elite Mapper**.
```

- ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661)
- ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768)

> ¹ ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) saat ini menyandang gelar **Elite Mapper II** atas keberhasilannya dalam memboyong gelar Elite Mapper pada dua kontes mapping yang berbeda, yakni [A Labour of Love](wiki/en/Contests/A_Labour_of_Love_Contest) dan [Monthly Beatmapping Contest](/wiki/en/Contests/Monthly_Beatmapping_Contest).

Reviewer 2: ```suggestion
¹ ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515) saat ini mempunyai gelar **Elite Mapper II** atas keberhasilannya dalam meraih gelar Elite Mapper pada dua kontes mapping yang berbeda, yakni [A Labour of Love](wiki/en/Contests/A_Labour_of_Love_Contest) dan [Monthly Beatmapping Contest](/wiki/en/Contests/Monthly_Beatmapping_Contest).
```

### Elite Nominator

> Sejak tahun 2020, setiap tahunnya osu! menganugerahkan gelar **Elite Nominator** kepada para anggota [Beatmap Nominator (BN)](/wiki/People/The_Team/Beatmap_Nominators) dan [Nomination Assessment Team (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) yang dinilai telah memberikan kontribusi yang lebih selama setahun ke belakang. Adapun gelar Elite Nominator tersebut akan hilang apabila pengguna yang bersangkutan dikeluarkan dan/atau mengundurkan diri dari BN atau NAT.

Reviewer 3: ```suggestion
Sejak tahun 2020, setiap tahunnya, osu! menganugerahkan gelar **Elite Nominator** kepada para anggota [Beatmap Nominator (BN)](/wiki/People/The_Team/Beatmap_Nominators) dan [Nomination Assessment Team (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) yang dinilai telah memberikan kontribusi yang lebih selama setahun ke belakang. Adapun gelar Elite Nominator tersebut akan hilang apabila pengguna yang bersangkutan dikeluarkan dan/atau mengundurkan diri dari BN atau NAT.
```

> Sejak tahun 2020, setiap tahunnya osu! menganugerahkan gelar **Elite Nominator** kepada para anggota [Beatmap Nominator (BN)](/wiki/People/The_Team/Beatmap_Nominators) dan [Nomination Assessment Team (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) yang dinilai telah memberikan kontribusi yang lebih selama setahun ke belakang. Adapun gelar Elite Nominator tersebut akan hilang apabila pengguna yang bersangkutan dikeluarkan dan/atau mengundurkan diri dari BN atau NAT.

Reviewer 3: ```suggestion
Sejak tahun 2020, setiap tahunnya, osu! menganugerahkan gelar **Elite Nominator** kepada para anggota [Beatmap Nominator (BN)](/wiki/People/The_Team/Beatmap_Nominators) dan [Nomination Assessment Team (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) yang dinilai telah memberikan kontribusi yang lebih selama setahun ke belakang. Adapun gelar Elite Nominator tersebut akan hilang apabila pengguna yang bersangkutan dikeluarkan dan/atau mengundurkan diri dari BN atau NAT.
```

Author: hmm, i'm actually quite certain that putting another comma there is linguistically speaking not preferred tho so i'll be leaving this sentence as is

applied the rest of the suggestions tho, ty for checking!

> Sejak tahun 2020, setiap tahunnya osu! menganugerahkan gelar **Elite Nominator** kepada para anggota [Beatmap Nominator (BN)](/wiki/People/The_Team/Beatmap_Nominators) dan [Nomination Assessment Team (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) yang dinilai telah memberikan kontribusi yang lebih selama setahun ke belakang. Adapun gelar Elite Nominator tersebut akan hilang apabila pengguna yang bersangkutan dikeluarkan dan/atau mengundurkan diri dari BN atau NAT.

Reviewer 2: ```suggestion
Sejak tahun 2020, setiap tahunnya osu! memberikan gelar **Elite Nominator** kepada para anggota [Beatmap Nominator (BN)](/wiki/People/The_Team/Beatmap_Nominators) dan [Nomination Assessment Team (NAT)](/wiki/People/The_Team/Nomination_Assessment_Team) yang dinilai telah memberikan kontribusi yang lebih selama setahun ke belakang. Adapun gelar Elite Nominator tersebut akan hilang apabila pengguna yang bersangkutan dikeluarkan dan/atau mengundurkan diri dari BN atau NAT.
```

- ![][flag_CA] [Agatsu](https://osu.ppy.sh/users/5579871)
- ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410)¹
- ![][flag_PH] [Jemzuu](https://osu.ppy.sh/users/7890134)¹
- ![][flag_DE] [Lasse](https://osu.ppy.sh/users/896613)¹
- ![][flag_DE] [Mordred](https://osu.ppy.sh/users/7265097)¹
- ![][flag_US] [Unpredictable](https://osu.ppy.sh/users/7560872)

> ¹ ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410), ![][flag_PH] [Jemzuu](https://osu.ppy.sh/users/7890134), ![][flag_DE] [Lasse](https://osu.ppy.sh/users/896613), dan ![][flag_DE] [Mordred](https://osu.ppy.sh/users/7265097) saat ini menyandang gelar **Elite Nominator II** atas keberhasilan mereka dalam mempertahankan status Elite Nominator yang mereka miliki selama dua tahun berturut-turut.

Reviewer 2: ```suggestion
¹ ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410), ![][flag_PH] [Jemzuu](https://osu.ppy.sh/users/7890134), ![][flag_DE] [Lasse](https://osu.ppy.sh/users/896613), dan ![][flag_DE] [Mordred](https://osu.ppy.sh/users/7265097) saat ini mempunyai gelar **Elite Nominator II** atas keberhasilan mereka dalam mempertahankan status Elite Nominator yang mereka miliki selama dua tahun berturut-turut.
```

### Pemenang kontes-kontes mapping Aspire

> osu! menganugerahkan gelar **Elite Mapper: Aspirant** kepada para pemenang kontes-kontes mapping Aspire.

Reviewer 2: ```suggestion
osu! memberikan gelar **Elite Mapper: Aspirant** kepada para pemenang kontes-kontes mapping Aspire.
```

- ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000)¹
- ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699)
- ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438)
- ![][flag_NL] [Mr HeliX](https://osu.ppy.sh/users/2330619)
- ![][flag_FI] [ProfessionalBox](https://osu.ppy.sh/users/3250792)
- ![][flag_TW] [SnowNiNo_](https://osu.ppy.sh/users/2506267)
- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)²
- ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231)

> ¹ ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) saat ini menyandang gelar **Elite Mapper II: Aspirant** atas keberhasilannya dalam menjuarai [Monthly Beatmapping Contest #1](https://osu.ppy.sh/home/news/2013-10-20-monthly-beatmapping-contest-1-results) dan [Monthly Beatmapping Contest: Aspire (2014)](https://osu.ppy.sh/home/news/2014-11-13-monthly-beatmapping-contest-aspire-results) secara bersamaan.

Reviewer 2: ```suggestion
¹ ![][flag_US] [Charles445](https://osu.ppy.sh/users/85000) saat ini mempunyai gelar **Elite Mapper II: Aspirant** atas keberhasilannya dalam menjuarai [Monthly Beatmapping Contest #1](https://osu.ppy.sh/home/news/2013-10-20-monthly-beatmapping-contest-1-results) dan [Monthly Beatmapping Contest: Aspire (2014)](https://osu.ppy.sh/home/news/2014-11-13-monthly-beatmapping-contest-aspire-results) secara bersamaan.
```

> ² Atas permintaan dari pihak yang bersangkutan, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini menyandang gelar **osu!catch Champion**.

Reviewer 3: ```suggestion
² Atas permintaannya, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini menyandang gelar **osu!catch Champion**.
```
kinda unnecessarily long imo

> ² Atas permintaan dari pihak yang bersangkutan, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini menyandang gelar **osu!catch Champion**.

Reviewer 3: ```suggestion
² Atas permintaannya, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini menyandang gelar **osu!catch Champion**.
```
kinda unnecessarily long imo

> ² Atas permintaan dari pihak yang bersangkutan, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini menyandang gelar **osu!catch Champion**.

Reviewer 2: ```suggestion
² Atas permintaan dari pihak yang bersangkutan, ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598) saat ini mempunyai gelar **osu!catch Champion**.
```

### Pemenang kontes-kontes osu!remix

> osu! menganugerahkan gelar **osu!mixer** kepada para pemenang kontes-kontes osu!remix.

Reviewer 2: ```suggestion
osu! memberikan gelar **osu!mixer** kepada para pemenang kontes-kontes osu!remix.
```

- ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471)¹
- ![][flag_FR] [Shinwoir](https://osu.ppy.sh/users/8984574)

> ¹ ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) saat ini menyandang gelar **osu!mixer II** atas keberhasilannya dalam menjuarai dua kontes osu!remix yang berbeda, yakni [osu!remix contest #1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) dan [A Labour of Love](wiki/en/Contests/A_Labour_of_Love_Contest).

Reviewer 2: ```suggestion
¹ ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) saat ini mempunyai gelar **osu!mixer II** atas keberhasilannya dalam menjuarai dua kontes osu!remix yang berbeda, yakni [osu!remix contest #1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) dan [A Labour of Love](wiki/en/Contests/A_Labour_of_Love_Contest).
```

### Featured Artist

> osu! menganugerahkan gelar **Featured Artist** kepada para [Featured Artist](/wiki/Featured_Artists) yang memiliki akun osu! sebagai salah satu bentuk balas jasa atas lagu-lagu yang telah mereka sediakan.

Reviewer 2: ```suggestion
osu! memberikan gelar **Featured Artist** kepada para [Featured Artist](/wiki/Featured_Artists) yang memiliki akun osu! sebagai salah satu bentuk balas jasa atas lagu-lagu yang telah mereka sediakan untuk osu!.
```

Author: similar to http://github.com/ppy/osu-wiki/pull/5622#discussion_r642110326 i don't think making the word `osu!` to be redundant here is a good thing :(

> ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) dan ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) saat ini menyandang gelar **osu!musician** atas kontribusi mereka dalam menata efek-efek suara dan menggubah lagu tema resmi yang dimiliki oleh osu!. Di sisi lain, ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) lebih memilih untuk mempertahankan gelar **osu!mixer** yang dimilikinya di saat ia diangkat menjadi Featured Artist.

Reviewer: ```suggestion
![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) dan ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) saat ini menyandang gelar **osu!musician** atas kontribusi mereka dalam menata efek-efek suara dan mengubah lagu tema resmi yang dimiliki oleh osu!. Di sisi lain, ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) lebih memilih untuk mempertahankan gelar **osu!mixer** yang dimilikinya di saat ia diangkat menjadi Featured Artist.
```

Author: oh, this is not a typo tho -- the is indeed meant to be `menggubah` instead of `mengubah`

- [Menggubah](http://kbbi.web.id/gubah) : "composing, (to) compose"
- Mengubah : "changing"

> ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) dan ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) saat ini menyandang gelar **osu!musician** atas kontribusi mereka dalam menata efek-efek suara dan menggubah lagu tema resmi yang dimiliki oleh osu!. Di sisi lain, ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) lebih memilih untuk mempertahankan gelar **osu!mixer** yang dimilikinya di saat ia diangkat menjadi Featured Artist.

Reviewer 2: ```suggestion
![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) dan ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102) saat ini mempunyai gelar **osu!musician** atas kontribusi mereka dalam menata efek-efek suara dan menggubah lagu tema resmi yang dimiliki oleh osu!. Di sisi lain, ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) lebih memilih untuk mempertahankan gelar **osu!mixer** yang dimilikinya ketika dia diangkat menjadi Featured Artist.
```

| ![][flag_RU] [tieff](https://osu.ppy.sh/users/89619) | [tieff](https://osu.ppy.sh/beatmaps/artists/34) |
| ![][flag_CA] [VINXIS](https://osu.ppy.sh/users/4323406) | [VINXIS](https://osu.ppy.sh/beatmaps/artists/22) |

### Pemenang kontes-kontes desain maskot

> Dari waktu ke waktu, osu! telah beberapa kali mengadakan kontes desain maskot yang terbuka bagi seluruh ilustrator yang tertarik untuk mendesain atau merombak tampilan maskot-maskot yang ada. osu! menganugerahkan gelar **osu!artist** kepada para ilustrator yang karyanya berhasil terpilih sebagai pemenang.

Reviewer 2: ```suggestion
Dari waktu ke waktu, osu! telah beberapa kali mengadakan kontes desain maskot yang terbuka bagi seluruh ilustrator yang tertarik untuk mendesain atau merombak tampilan maskot-maskot yang ada. osu! memberikan gelar **osu!artist** kepada para ilustrator yang karyanya berhasil terpilih sebagai pemenang.
```

- ![][flag_US] [Crowie](https://osu.ppy.sh/users/6894067) mendesain **Mocha** dan berbagai aset grafis lainnya yang ada di osu!.
- ![][flag_ID] [JMC](https://osu.ppy.sh/users/774010) memenangkan kontes [Fan Art 6: osu!taiko mascot](https://osu.ppy.sh/community/contests/2).
- ![][flag_US] [Thievley](https://osu.ppy.sh/users/4717672)  merombak ulang desain **Yuzu**.
- ![][flag_ID] [xiemon](https://osu.ppy.sh/users/5203667) mendesain **Mani and Mari**.

### Hall of Famer

> osu! menganugerahkan gelar **osu! Hall of Famer** kepada tiga nama di bawah ini atas kontribusinya masing-masing dalam menjunjung tinggi standar permainan pada masa-masa awal berdirinya osu!.

Reviewer 2: ```suggestion
osu! memberikan gelar **osu! Hall of Famer** kepada tiga nama di bawah ini, atas kontribusinya masing-masing dalam menjunjung tinggi standar permainan pada masa-masa awal berdirinya osu!.
```

<!-- TODO: reasons -->

- ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589)
- ![][flag_PH] [James](https://osu.ppy.sh/users/5728)
- ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781)

### Elite Storyboarder

> osu! menganugerahkan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh khalayak luas.

Reviewer 2: ```suggestion
osu! memberikan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh banyak pemain.
```

Author: not really a fan of how `banyak` is being repeated like that in a single sentence here (`...banyak karya yang dikagumi oleh banyak...`) so i think i'll be leaving this as is :(

> osu! menganugerahkan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh khalayak luas.

Reviewer 3: ```suggestion
osu! menganugerahkan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh khalayak ramai.
```

> osu! menganugerahkan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh khalayak luas.

Reviewer 3: ```suggestion
osu! menganugerahkan gelar **Elite Storyboarder** kepada para pembuat storyboard (*storyboarder*) yang telah menciptakan banyak karya yang dikagumi oleh khalayak ramai.
```

- ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231)
- ![][flag_CN] [yf_bmp](https://osu.ppy.sh/users/1243669)

### Terkait dengan permainan

> Pemain-pemain berikut memperoleh gelarnya masing-masing atas keberhasilan mereka dalam menorehkan rekor-rekor yang luar biasa atau menyelesaikan tantangan-tantangan tertentu yang terkait dengan permainan.

Reviewer 2: ```suggestion
Pemain-pemain berikut di bawah ini memperoleh gelarnya masing-masing atas keberhasilan mereka dalam memecahkan rekor-rekor yang luar biasa atau menyelesaikan tantangan-tantangan tertentu yang terkait dengan permainan.
```

| Pengguna | Gelar | Pencapaian |
| :-- | :-- | :-- |

> | ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | Menorehkan FC NoMod pertama pada beatmap [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |

Reviewer 2: ```suggestion
| ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581) | Champion Above Champions | Mendapatkan FC No-Mod pertama pada beatmap [Shotgun Symphony+ [Impossibly Intense]](https://osu.ppy.sh/beatmapsets/7671#osu/32570) |
```

Author: o you don't spell `NoMod` as `No-Mod` orz. what is currently in the page is already correct :(

![image](https://user-images.githubusercontent.com/36564236/120115046-28eba680-c1ac-11eb-9070-da15f1cabd64.png)

> | ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Menorehkan FC pada semua beatmap osu!taiko yang berstatus Ranked per tanggal 3 November 2019 |

Reviewer 2: ```suggestion
| ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352) | Drum Decimator | Mendapatkan FC pada semua beatmap osu!taiko yang berstatus Ranked sampai tanggal 3 November 2019 |
```

| ![][flag_NL] [Shirotora_old](https://osu.ppy.sh/users/11820) | Bested the Reaper's Game | *Tidak terdokumentasi* <!-- TODO: a reference to TWEWY's 104 building, because he got to rank #104 in a certain amt of time.. need to investigate further --> |

> | ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Memainkan semua beatmap osu! yang berstatus Ranked per tanggal 27 April 2019 |

Reviewer 2: ```suggestion
| ![][flag_US] [xasuma](https://osu.ppy.sh/users/3172980) | The First Completionist | Memainkan semua beatmap osu! yang berstatus Ranked sampai tanggal 27 April 2019 |
```
